hydra-node/src/Hydra/Node.hs

```diff
@@ -310,9 +310,13 @@ stepHydraNode node = do
 
   HydraNode{tracer, inputQueue = InputQueue{dequeue, reenqueue}, env} = node
 
--- | The time to wait between re-enqueuing a 'Wait' outcome from 'HeadLogic'.
+-- | The maximum number of times to re-enqueue a 'Wait' outcome.
+defaultTTL :: TTL
+defaultTTL = 10000
+
+-- | The time to wait between re-enqueuing a 'Wait' outcome
 waitDelay :: DiffTime
-waitDelay = 0.1
+waitDelay = 1
```

Comment:
This is not good, we should _at least_ make this wait configurable different for waits "on L2 stuff" vs. the wait for deposits to become active.

However, we should address the situation of protocol logic needing to wait in the first place!